onnx/test/symbolic_shape_test.py

```diff
@@ -104,9 +104,8 @@ def test_two_symbolic_concat(self):  # type: () -> None
         inferred_model = onnx.shape_inference.infer_shapes(onnx_model, strict_mode=True)
         self._assert_valueinfo_shape(inferred_model, [
             make_tensor_value_info("C", TensorProto.FLOAT, (2, -1)),
-            make_tensor_value_info("E", TensorProto.FLOAT, (2, -1))])
-        # the symbolic shape of E and output should be the same
-        assert self._get_shape_from_name(inferred_model, 'E') == self._get_shape_from_name(inferred_model, 'output')
+            make_tensor_value_info("E", TensorProto.FLOAT, (2, -1)),
```

Comment:
Maybe it would be better to change line 100 above and replace 'M' by None? And leave the current check?

Reply:
Ditto. Done!